lib/storage/metadata/mongoclient/LogConsumer.js

```diff
@@ -18,12 +17,9 @@ class LogConsumer {
      */
     constructor(mongoConfig, logger) {
         const { replicaSetHosts, database } = mongoConfig;
-        // 'local' is the database where MongoDB has oplogs.rs capped collection
-        this.database = 'local';
-        this.mongoUrl = `mongodb://${replicaSetHosts}/local`;
-        this.logger = logger;
-        this.metadataDatabase = database;
-        this.oplogNsRegExp = new RegExp(`^${database}\\.`);
+        this._mongoUrl = `mongodb://${replicaSetHosts}/local`;
+        this._logger = logger;
+        this._oplogNsRegExp = new RegExp(`^${database}\\.`);
```

Comment:
what does it bring to underscore mongoUrl, logger and oplogNsRegExp ? because it makes a bigger changeset, which is therefore harder to read.

Reply:
They are private variable, to mark them as such (per coding guidelines we use) they should start with an underscore